website/src/pages/en/subgraphs/cookbook/subgraph-composition-three-sources.mdx

```diff
@@ -0,0 +1,98 @@
+> Important Reminders:
+>
+> - Subgraph composition is built into the CLI, and you can deploy with [Subgraph Studio](https://thegraph.com/studio/).
+> - This feature requires `specVersion` 1.3.0.
+
+## Overview
+
+Subgraph composition empowers you use one Subgraph as a data source for another, allowing it to consume and respond to entity changes. Instead of fetching on-chain data directly, a Subgraph can listen for updates from another Subgraph and react to changes. This is useful for aggregating data from multiple Subgraphs or triggering actions based on external updates.
+
+## Prerequisites
+
+To deploy **all** Subgraphs locally, you must have the following:
+
+- A [graph-node](https://github.com/graphprotocol/graph-node) instance running locally
+- An [IPFS](https://docs.ipfs.tech/) instance running locally
+- [Node.js](https://nodejs.org) and npm
+
+## Get Started
+
+The following guide provides examples for defining three source Subgraphs to create one powerful composed Subgraph.
+
+### Specifics
+
+- To keep this example simple, all source Subgraphs use only block handlers. However, in a real environment, each source Subgraph will use data from different smart contracts.
+- The examples below show how to import and extend the schema of another Subgraph to enhance its functionality.
+- Each source Subgraph is optimized with a specific entity.
+- All the commands listed install the necessary dependencies, generate code based on the GraphQL schema, build the Subgraph, and deploy it to your local Graph Node instance.
+
+### Step 1. Deploy Block Time Source Subgraph
+
+This first source Subgraph calculates the block time for each block.
+
+- It imports schemas from other Subgraphs and adds a `block` entity with a `timestamp` field, representing the time each block was mined.
+- It listens to time-related blockchain events (e.g., block timestamps) and processes this data to update the Subgraph's entities accordingly.
+
+To deploy this Subgraph locally, run the following commands:
+
+```bash
+npm install
+npm run codegen
+npm run build
+npm run create-local
+npm run deploy-local
+```
+
+### Step 2. Deploy Block Cost Source Subgraph
+
+This second source Subgraph indexes the cost of each block.
+
+#### Key Functions
+
+- It imports schemas from other Subgraphs and adds a `block` entity with cost-related fields.
+- It listens to blockchain events related to costs (e.g. gas fees, transaction costs) and processes this data to update the Subgraph's entities accordingly.
+
+To deploy this Subgraph locally, run the same commands as above.
+
+### Step 3. Define Block Size in Source Subgraph
+
+This third source Subgraph indexes the size of each block.
+
+#### Key Functions
+
+- It imports existing schemas from other Subgraphs and adds a `block` entity with a `size` field representing each block's size.
+- It listens to blockchain events related to block sizes (e.g., storage or volume) and processes this data to update the Subgraph's entities accordingly.
```

Comment:
I guess this one doesn't need to be deployed locally? (It doesn't say "To deploy this Subgraph locally, run the same commands as above")

Reply:
Same for the 4th one?